gridcomps/History/Sampler/MAPL_GeosatMaskMod.F90

```diff
@@ -0,0 +1,167 @@
+module MaskSamplerGeosatMod
```

Comment:
Please change all "GeoSat" to "GeoStat".   (This includes renaming the file itself.)